### PR TITLE
Feature/display talents from db (WIP)

### PR DESCRIPTION
**Pull request by LittleSweetGun**
TODO: Update talents-result so the cards will show the profils from the DB

Update goodhive.users table:
CREATE TABLE goodhive.users (
  id SERIAL PRIMARY KEY,
  title VARCHAR(100) NOT NULL,
  job_headline VARCHAR(255) NOT NULL,
  first_name VARCHAR(100) NOT NULL,
  last_name VARCHAR(100) NOT NULL,
  country VARCHAR(100) NOT NULL,
  city VARCHAR(100) NOT NULL,
  phone_country_code VARCHAR(10) NOT NULL,
  phone_number VARCHAR(20) NOT NULL,
  email VARCHAR(255) NOT NULL UNIQUE,
  telegram VARCHAR(255),
  currency VARCHAR(100),
  rate VARCHAR(20),
  about_work TEXT NOT NULL,
  skills VARCHAR(255) NOT NULL,
  image_url VARCHAR (255)
);

### CODE REVIEW

**Comment by LittleSweetGun** [comment]
TODO:
- Display talents image
- Redesign card and make it responsive

File: app/api/companies/search-talents/route.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    rejectUnauthorized: false, // This allows connecting to a database with a self-signed certificate
+  },
+});
+
+export async function GET(request: Request) {
+  {
+    // Fetch data from the database or any external API
+    // Return the fetched data as a response
+    try {
+      const data = await sql`SELECT * FROM goodhive.users`;
```
Comment (n1c01a5) [inline app/api/companies/search-talents/route.ts:14]: `users`

File: app/api/companies/search-talents/route.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+const sql = postgres(process.env.DATABASE_URL || "", {
+  ssl: {
+    rejectUnauthorized: false, // This allows connecting to a database with a self-signed certificate
+  },
+});
+
+export async function GET(request: Request) {
+  {
+    // Fetch data from the database or any external API
+    // Return the fetched data as a response
```
Comment (n1c01a5) [inline app/api/companies/search-talents/route.ts:12]: Don't put obvious comment.

File: app/companies/search-talents/page.tsx [hunk 3]
```diff
@@ -1,21 +1,55 @@
-import { Metadata } from "next";
+"use client";
+
+import { useEffect, useState } from "react";
 import Header from "@/app/components/header";
```
Comment (n1c01a5) [inline app/companies/search-talents/page.tsx:4]: Add break line between react and other imports.

**Review by n1c01a5 — CHANGES_REQUESTED** [review]

**Comment by LittleSweetGun** [comment]
Add currency in job_offers table

ALTER TABLE goodhive.job_offers
ADD currency VARCHAR(100)

**Comment by LittleSweetGun** [comment]
#80 #84 #83

**Pull request merged**